src/theme/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { createMuiTheme, Theme, styled, useTheme } from '@material-ui/core'
+import { unstable_createMuiStrictModeTheme as createMuiTheme, Theme, useTheme } from '@material-ui/core'
```

Comment:
For dealing with the warning of Strict mode. 
https://stackoverflow.com/questions/61220424/material-ui-drawer-finddomnode-is-deprecated-in-strictmode